Listenbrainz/UI/Screens/PlayerView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  PlayerView.swift
+//  Listenbrainz
+//
+//  Created by Gaurav Bhardwaj on 16/04/23.
+//
+
+
+import SwiftUI
+
+struct PlayerView : View{
+    
+    var body: some View{
+        
+        Text("Player")
```

Comment:
For next PRs, try making a Constants file and add the font styles, colors to it so that we can provide those to the texts and keep it consitent across app.

Reply:
sure :)